lib/dependabot/file_parsers/docker/docker.rb

```diff
@@ -0,0 +1,52 @@
+# frozen_string_literal: true
+
+require "dependabot/dependency"
+require "dependabot/file_parsers/base"
+
+module Dependabot
+  module FileParsers
+    module Docker
+      class Docker < Dependabot::FileParsers::Base
+        PATH = /[a-z0-9]+(?:[._-][a-z0-9]+)*/
+        IMAGE = %r{(?<image>#{PATH}(?:/#{PATH})*)}
+        TAG = /:(?<tag>[a-zA-Z0-9_]+[A-z0-9._-]*)/
+        DIGEST = /@(?<digest>[0-9a-f]+)/
+        NAME = / AS (?<name>[a-zA-Z0-9_-]+)/
+        FROM_LINE = /^FROM #{IMAGE}#{TAG}?#{DIGEST}?#{NAME}?/
```

Comment:
Convention is `FROM` but spec allows `from` (https://docs.docker.com/engine/reference/builder/#format)

Reply:
Awesome, have added support for nutcases...!